go.mod

```diff
@@ -203,6 +203,9 @@ replace github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
 // allows us to specify that as an option.
 replace google.golang.org/protobuf => github.com/lightninglabs/protobuf-go-hex-display v1.30.0-hex-display
 
+// Temporary replace until the next version of sqldb is tagged.
+replace github.com/lightningnetwork/lnd/sqldb => ./sqldb
```

Comment:
cool - not sure we want to merge with this thought right? 

So I think might need to open separate PR that points to this (so we can verify complete CI run) and then once this PR is merged and tagged, then update that PR to point to the tagged version

Reply:
So the next two migration PRs both touch sqldb and other packages depending on it so i think we can't avoid. Maybe can move this to the next PR (#8855) though.